modules.json

```diff
@@ -326,10 +334,16 @@
   "rxjs": {
     "var": "Rx",
     "versions": {
-      "*": {
+      "< 6.0.0": {
         "development": "https://unpkg.com/rxjs@[version]/bundles/Rx.js",
         "production": "https://unpkg.com/rxjs@[version]/bundles/Rx.min.js"
       }
+    },
+    "$disabled": {
```

Comment:
what is that?

Reply:
json has no comments. rxjs 6+ has different var name. it's not supported by your modules. I wanted to make some PR later, after ci start working without fail. also I wanted to make sure that everyone can see that it makes no sense to add 6+.

Reply:
ok, I see do you mind replacing it with a comment and use https://github.com/sindresorhus/strip-json-comments
```diff
- const modules = require('./modules');
+ const modules = stripJsonComments(fs.readFileSync(path.join(__dirname, './modules')))
```

Reply:
it's not a blocker, I could take care of that, I will merge your PR by tomorrow regardless